index.ts

```diff
@@ -32,6 +32,8 @@ async function shouldRunMigrations(): Promise<boolean> {
 	});
 }
 
+export type StorageType = 'sync' | 'local' | 'managed';
```

Comment:
`managed` is read-only, it's not supported by `webext-options-sync`
```suggestion
export type StorageType = 'sync' | 'local';
```